src/Npgsql/NpgsqlSchema.cs

```diff
@@ -16,33 +16,38 @@ namespace Npgsql
     static class NpgsqlSchema
     {
         public static DataTable GetSchema(NpgsqlConnection conn, string? collectionName, string?[]? restrictions)
+            => GetSchemaAsync(conn, collectionName, restrictions, false).GetAwaiter().GetResult();
```

Comment:
This method isn't needed - call directly into the one below and pass `async: false` as usualy